src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -445,6 +445,9 @@ fn make_data(
             data.insert("playpen_line_numbers".to_owned(), json!(true));
         }
     }
+    if html.playpen.copyable {
```

Comment:
It should be `html_config`.

```suggestion
    if html_config.playpen.copyable {
```